Classes/Util/PBOpenFiles.h

```diff
@@ -8,9 +8,11 @@
 
 #import <Foundation/Foundation.h>
 
+@class PBChangedFile;
+
 @interface PBOpenFiles : NSObject
 
-+ (void)showInFinderAction:(id)sender with:(NSURL *)workingDirectoryURL;
-+ (void)openFilesAction:(id)sender with:(NSURL *)workingDirectoryURL;
++ (void)openFiles:(NSArray<PBChangedFile *> *)selectedFiles with:(NSURL *)workingDirectoryURL;
```

Comment:
That code was originally in `PBGitRepositoryDocument`, where the responder chain would get to it in some way. And some things can throw paths at it (like, right-click-opening a changed file in the history view commit pane), which is why I wrote it that way. Now that we have submodules, we have that (generic) method, the stuff you've added for submodules, and a bug because right-click-open in the history view doesn't do the same thing...